src/disks.py

```diff
@@ -0,0 +1,98 @@
+from logging import getLogger
+
+import dbus
+from _dbus_bindings import PROPERTIES_IFACE
+
+logger = getLogger("yunohost.storage")
+
+
+__all__ = ["info", "list"]
+
+
+UDISK_DRIVE_PATH = "/org/freedesktop/UDisks2/drives/"
+UDISK_DRIVE_IFC = "org.freedesktop.UDisks2.Drive"
+
+
+def _humaize(byte_size):
```

Comment:
Maybe you should use this function instead ?
https://github.com/YunoHost/yunohost/blob/4e9d94c836fa34b37805bf07b61aa8aa0d9d61e3/src/utils/system.py#L120